script/elevate-cpanel.PL

```diff
@@ -609,12 +620,30 @@ EOS
     say <<EOS;
 This is your chance now to cancel the update process.
 You can hit CTRL-C to abort, otherwise the install starts in a few seconds.
```

Comment:
Lines 621 and 622 no longer make sense after this update

Reply:
If we remove 621 and 622, and, 628-630, then, won't 624, and, 625 scroll right past them?